src/app/location-selection/location-selection.component.html

```diff
@@ -92,86 +99,95 @@
-          class="criteria-spinner"
-          mode="indeterminate"
-          diameter="32"
-          [matTooltip]="getLoadingRegionsMessage(busyRegions)"
+        <button
+          mat-mini-fab
+          (click)="mapService.toggleEditor()"
+          matTooltipPosition="right"
+          matTooltip="Notes"
         >
-        </mat-progress-spinner>
+          <mat-icon>edit_note</mat-icon>
+        </button>
+      </div>
+      <div class="center-container">
+        @let progress = mapService.progress$ | async;
+        @if (progress != null) {
+          <mat-progress-bar
+            class="layer-progress"
+            [value]="progress * 100"
+            [style.visibility]="progress === 1 ? 'hidden' : 'visible'"
+          >
+          </mat-progress-bar>
         }
       </div>
       <button
+        class="profile-button"
         mat-mini-fab
-        (click)="openDrawer('style')"
-        matTooltipPosition="right"
-        matTooltip="Layer Style"
-      >
-        <mat-icon>brush</mat-icon>
-      </button>
-      <button
-        mat-mini-fab
-        (click)="mapService.toggleEditor()"
-        matTooltipPosition="right"
-        matTooltip="Notes"
-      >
-        <mat-icon>edit_note</mat-icon>
-      </button>
-    </div>
-    <div class="center-container">
-      @let progress = mapService.progress$ | async; @if (progress != null) {
-      <mat-progress-bar
-        class="layer-progress"
-        [value]="progress * 100"
-        [style.visibility]="progress === 1 ? 'hidden' : 'visible'"
-      >
-      </mat-progress-bar>
-      }
-    </div>
-
-    <button
-      class="profile-button"
-      mat-mini-fab
-      [matMenuTriggerFor]="profileMenu"
-      [matTooltip]="user?.email"
-      matTooltipPosition="left"
-    >
-      <mat-icon>account_circle</mat-icon>
-    </button>
-    <mat-menu #profileMenu="matMenu">
-      <button mat-menu-item (click)="openConfig()">
-        <mat-icon>settings</mat-icon>
-        <span>Settings</span>
-      </button>
-      <button
-        mat-menu-item
-        *ngIf="user?.roles?.includes('ADMIN')"
-        (click)="openClusterAdmin()"
+        [matMenuTriggerFor]="profileMenu"
+        [matTooltip]="user?.email"
+        matTooltipPosition="left"
       >
-        <mat-icon>dns</mat-icon>
-        <span>Manage Cluster</span>
-      </button>
-      <button mat-menu-item (click)="authService.logout()">
-        <mat-icon>logout</mat-icon>
-        <span>Logout</span>
+        <mat-icon>account_circle</mat-icon>
       </button>
-    </mat-menu>
+      <mat-menu #profileMenu="matMenu">
+        <button mat-menu-item (click)="openConfig()">
+          <mat-icon>settings</mat-icon>
+          <span>Settings</span>
+        </button>
+        <button mat-menu-item (click)="authService.logout()">
+          <mat-icon>logout</mat-icon>
+          <span>Logout</span>
+        </button>
+        <button
+          mat-menu-item
+          *ngIf="authService.isAdmin() | async"
```

Comment:
I'd probably wrap both these in an if, or use `@let` so you're not repeating the expression. Though idk if Angular is smart enough to see that it's the same expression and only subscribe once.